modules/node-observability-run-profiling-query.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="running-profiling-query_{context}"]
+= Running profiling query
+
+Profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. The Node Observability Operator stores the profiling data inside the container file system `/run/node-observability` directory. To request profiling data query, you have to create a `NodeObservabilityRun` resource.
+
+[IMPORTANT]
+====
+You can request only one profiling query at any point of time.
+====
+
+.Prerequisites
+* The Node Observability Operator has been installed.
```

Comment:
```suggestion
* You have installed the Node Observability Operator.
```
I noticed in your Prerequisites sections, you shift from active to passive voice. I suggest staying with active voice

Reply:
Thanks for pointing out. I have fixed it.